bentoml/_internal/io_descriptors/file.py

```diff
@@ -8,6 +8,7 @@
 
 from .base import IODescriptor
 from ..types import FileLike
+from ..utils.http import finalize_http_response
```

Comment:
should we call this function with a different name? something like `set_headers` so that we don't confuse ourself with `finalize_http_response` from each of the io descriptors?

Reply:
Yeah, I'll rename it to set_content_length; that's what it does anyway.